requirements.txt

```diff
@@ -1,3 +1,6 @@
+numpy
```

Comment:
You should add the requirements to the `setup.py`: https://github.com/RealVNF/coord-env-interface/blob/master/setup.py#L4